frontend/controller/router.js

```diff
@@ -87,7 +87,7 @@ const router = new Router({
     },
     {
       path: '/design-system',
-      component: () => DesignSystem,
+      component: DesignSystem,
```

Comment:
I believe this change needs to be reverted by merging the recent fixes in `master` (see @SebinSong's recent fix for this: #1011).

Reply:
Somehow this change was added back. Can you revert it again?